sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/CanonicalizeSuite.scala

```diff
@@ -95,4 +96,19 @@ class CanonicalizeSuite extends SparkFunSuite {
     val castWithTimeZoneId = Cast(literal, LongType, Some(TimeZone.getDefault.getID))
     assert(castWithTimeZoneId.semanticEquals(cast))
   }
+
+  test("SPARK-32927: Bitwise operations are commutative") {
+    Seq(
```

Comment:
nit:

```scala
    Seq(BitwiseOr(_, _), BitwiseAnd(_, _), BitwiseXor(_, _)).foreach { f =>
      val e1 = f('a, f('b, 'c))
      val e2 = f(f('a, 'b), 'c)
      val e3 = f('a, f('b, 'a))

      assert(e1.canonicalized == e2.canonicalized)
      assert(e1.canonicalized != e3.canonicalized)
    }
```

Reply:
Oh, didn't know that syntax, thank you, will change.